hack/podmansnoop

```diff
@@ -125,7 +125,12 @@ def _print_event(cpu, data, size): # callback
 
     comm = e.comm.decode()
     if comm == "3":
-        # For absolutely unknown reasons, 'crun' appears as '3'.
+        # Because of CVE-2019-5736, crun copies itself on a memfd or temp file, add seals,
+        # then goes fexecve. The linux kernel will then set comm as the basename of
+        # /dev/fd/<fdnum>, which happens to be 3 being the first available file descriptor.
```

Comment:
See also https://github.com/torvalds/linux/blob/master/fs/exec.c#L1600 and https://github.com/torvalds/linux/blob/master/fs/exec.c#L1425